build/rabbitmq/scripts/docker-entrypoint-wrapper.sh

```diff
@@ -17,7 +17,7 @@ if [ ! -e $LOCK_FILE ]; then
 fi
 
 if [ $# -eq 0 ]; then
-  exec /docker-entrypoint.sh rabbitmq-server
+  exec docker-entrypoint.sh rabbitmq-server
 else
-  exec /docker-entrypoint.sh $@
+  exec docker-entrypoint.sh $@
```

Comment:
Not sure what 255d22ca41a254d4bc95488052f652a72c6df758 is addressing since current working directory is `/`.

Reply:
This did not work for me. Rabbitmq couldn't find the script.

Reply:
Is `/set-config.sh` correctly found then?

Reply:
Yes, cause we copied this into the root.

````
 docker build . -t test
[sudo] password for hotpi:
[+] Building 23.8s (7/7) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                                                                     0.1s
 => => transferring dockerfile: 187B                                                                                                                                                                     0.0s
 => [internal] load .dockerignore                                                                                                                                                                        0.1s
 => => transferring context: 2B                                                                                                                                                                          0.0s
 => [internal] load metadata for docker.io/library/rabbitmq:3.6.16-management                                                                                                                            2.1s
 => [internal] load build context                                                                                                                                                                        0.1s
 => => transferring context: 1.28kB                                                                                                                                                                      0.0s
 => [1/2] FROM docker.io/library/rabbitmq:3.6.16-management@sha256:05bd722c6b0c747ae7f854baa9e9ceb9b47da75e39f93ad6a1db74f788b0f828                                                                     20.2s
 => => resolve docker.io/library/rabbitmq:3.6.16-management@sha256:05bd722c6b0c747ae7f854baa9e9ceb9b47da75e39f93ad6a1db74f788b0f828                                                                      0.0s
 => => sha256:9df8e9792ce67836bd5fb07d0d47e75314d0b5586e15d2a7fce0361e3577f86d 9.72kB / 9.72kB                                                                                                           0.0s
 => => sha256:02fe1bd1a85c93837c66dbd7406515fcd5b8a683d0642581e8f3a54eb35ef173 4.50MB / 4.50MB                                                                                                           3.0s
 => => sha256:f17d81b4b692f7e0d6c1176c86b81d9f2cb5ac5349703adca51c61debcfe413c 22.49MB / 22.49MB                                                                                                        11.0s
 => => sha256:66c15a50f4da989e3d3c86925f82bec705c5c25346b92aad555a7b0da8efe7b8 4.07kB / 4.07kB                                                                                                           0.7s
 => => sha256:05bd722c6b0c747ae7f854baa9e9ceb9b47da75e39f93ad6a1db74f788b0f828 2.37kB / 2.37kB                                                                                                           0.0s
 => => sha256:eada5538ca4668b81e55cb31a2eb256d9c752775e99cb256efcf1db06107d98a 3.04kB / 3.04kB                                                                                                           0.0s
 => => sha256:771c4c62018c862476441914cb38dbf2c3fa9876effa266d27fc831ade13ff8e 951.87kB / 951.87kB                                                                                                       1.7s
 => => sha256:0ae9c44cefc970d06e67e34323059f924232f944d238090c834061cd9114c6b0 27.71MB / 27.71MB                                                                                                        13.5s
 => => sha256:638d18f9679b9ccd53ce56a192a94f7a3c8ed00bcc530da0e400db720fa1cd94 7.31MB / 7.31MB                                                                                                           7.3s
 => => sha256:80e59581508db919ce2b27b4df6c50514e9c6caa470e3761b17874d84d49ffeb 2.34kB / 2.34kB                                                                                                           7.6s
 => => sha256:065f8960e5370afaafc1a02bd11e8c9d35013110701ea8ed2488ecb9c3d25d1b 148B / 148B                                                                                                               7.9s
 => => sha256:91bf444c375323df1b1c7c6e6de873ccedd7cd25acad2fb9acd413f148966104 126B / 126B                                                                                                               8.2s
 => => sha256:6b34701157a4e042113a66cf8985972f74a7e8ad93521e81e43a481aabf5b788 4.28kB / 4.28kB                                                                                                           8.6s
 => => sha256:fafd1bae2225fc6ba40d093417aa97340ade378c17641c50bd5d9f260ab235a8 121B / 121B                                                                                                               9.0s
 => => sha256:2a87cf263e6d4a15b9623183a76e20fb8d906e39a82274205207d6136f7e195d 191B / 191B                                                                                                               9.3s
 => => sha256:d2806ea66b21c7b968494b790fed1fe7f8a56fbf2000ed9d973acb74e1754e40 7.64MB / 7.64MB                                                                                                          12.9s
 => => extracting sha256:f17d81b4b692f7e0d6c1176c86b81d9f2cb5ac5349703adca51c61debcfe413c                                                                                                                2.5s
 => => extracting sha256:02fe1bd1a85c93837c66dbd7406515fcd5b8a683d0642581e8f3a54eb35ef173                                                                                                                0.5s
 => => extracting sha256:66c15a50f4da989e3d3c86925f82bec705c5c25346b92aad555a7b0da8efe7b8                                                                                                                0.2s
 => => extracting sha256:771c4c62018c862476441914cb38dbf2c3fa9876effa266d27fc831ade13ff8e                                                                                                                0.2s
 => => extracting sha256:0ae9c44cefc970d06e67e34323059f924232f944d238090c834061cd9114c6b0                                                                                                                1.5s
 => => extracting sha256:638d18f9679b9ccd53ce56a192a94f7a3c8ed00bcc530da0e400db720fa1cd94                                                                                                                0.4s
 => => extracting sha256:80e59581508db919ce2b27b4df6c50514e9c6caa470e3761b17874d84d49ffeb                                                                                                                0.0s
 => => extracting sha256:065f8960e5370afaafc1a02bd11e8c9d35013110701ea8ed2488ecb9c3d25d1b                                                                                                                0.0s
 => => extracting sha256:91bf444c375323df1b1c7c6e6de873ccedd7cd25acad2fb9acd413f148966104                                                                                                                0.0s
 => => extracting sha256:6b34701157a4e042113a66cf8985972f74a7e8ad93521e81e43a481aabf5b788                                                                                                                0.0s
 => => extracting sha256:fafd1bae2225fc6ba40d093417aa97340ade378c17641c50bd5d9f260ab235a8                                                                                                                0.0s
 => => extracting sha256:2a87cf263e6d4a15b9623183a76e20fb8d906e39a82274205207d6136f7e195d                                                                                                                0.0s
 => => extracting sha256:d2806ea66b21c7b968494b790fed1fe7f8a56fbf2000ed9d973acb74e1754e40                                                                                                                0.9s
 => [2/2] COPY [ scripts/*, / ]                                                                                                                                                                          1.1s
 => exporting to image                                                                                                                                                                                   0.2s
 => => exporting layers                                                                                                                                                                                  0.1s
 => => writing image sha256:68b7cddc8cdf9f6f8ce42f90d813cca4e40bdf53ca9e234c9967e871c6470127                                                                                                             0.0s
 => => naming to docker.io/library/test                                                                                                                                                                  0.0s

docker run test /set-config.sh

/set-config.sh: Waiting for rabbitmq-server ...

/set-config.sh: Waiting for rabbitmq-server ...
^C^C^C^C
~/src/musicbrainz-docker/build/rabbitmq docker run test ls /
bin  boot  dev  docker-entrypoint-wrapper.sh  docker-entrypoint.sh  etc  home  lib  lib64  media  mnt  opt  plugins  proc  root  run  sbin  set-config.once  set-config.sh  srv  sys  tmp  usr  var
````